### PR TITLE
Add documentation for skygear event

### PR DESCRIPTION
**Pull request by ben181231**
connect SkygearIO/skygear-server#199

### CODE REVIEW

**Comment by oursky-travis** [comment]
@ben181231, thanks for your PR! By analyzing the annotation information on this pull request, we identified @bensonby, @cheungpat and @rickmak to be potential reviewers

File: content/plugin/guide-events.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+after all plugins finish handling `before-plugins-ready` event. Plugins are
+expected to insert seed data here if necessary.
+
+You can need to login to a worker user before inserting seed data.
+
+```python
+import skygear
+from skygear.options import options as skygear_options
+
+WORKDER_USERNAME = # get your worker username
```
Comment (bensonby) [inline content/plugin/guide-events.md]: s/WORKDER/WORKER

Comment (ben181231) [inline content/plugin/guide-events.md]: 😓

**Comment by ben181231** [comment]
@bensonby Updated.

**Comment by bensonby** [comment]
Picked as 00997ce.

**Pull request closed without merging**